safe-global/safe-wallet-web
# Separate watchlist from owned safes

Separate the list of safes on the my accounts page into owned safes and watchlist safes

Owned Safes includes:
- Safes that are both owned and added
- Safes that are owned but not added

Watchlist includes:
- safes that are not owned but added

## How to test it
- go to `/welcome/accounts`
- see the lists separated as described.

## Screenshots
<img width="778" alt="image" src="https://github.com/safe-global/safe-wallet-web/assets/17801424/a9bfbb1a-9e29-42a0-afda-0e93723b630a">


## Checklist
